projects/artstor-storage/src/lib/artstor-storage.service.ts

```diff
@@ -0,0 +1,120 @@
+import { Injectable, Inject, PLATFORM_ID } from '@angular/core';
+import { isDefined } from '@angular/compiler/src/util';
+import { localStorageFactory } from '@ng-toolkit/universal';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ArtstorStorageService {
+  constructor(@Inject(PLATFORM_ID) private platformId: Object) {}
+
+  // Is this code being interpretted by a browser-based client?
+  private isBrowser: boolean = this.platformId === 'browser'
```

Comment:
Angular does provide isPlatformBrowser(platformId) https://angular.io/api/common/isPlatformBrowser, which I think  we should use consistently, as:
`private isBrowser: boolean = isPlatformBrowser(this.platformId)`

Interestingly, using a simple string comparison should work just fine. PLATFORM_ID should always provide a string output of its token. The only case where it would fail is if in the future we wanted to use service workers, "isPlatformBrowser" would return true for ids of the browser runtime and browser service workers.